### ISSUE TITLE
Matrix constructor for column vector literals.

### ISSUE THREAD

**Issue by aishfenton** [scalanlp/breeze#550]
Maybe there's something I'm missing, but I don't think there's a constructor for making DenseMatrices from col vectors (but there is for row vectors, which seems a little counter intuitive since Breeze is col major). 

Ideally constructor would treat Transpose[DenseVector]* as row vectors and normal vectors as cols and support both.

File under: enhancement request. I might try and put together a pull request if time permits, if you people agree with it. Although this would be a change to existing semantics which could be weird for people.

**Comment by dlwh** [comment]
yeah, there's nothing. I don't feel safe changing the apply() method at this point. Breeze is used in enough places that breaking would cause major problems. Maybe fromColumns?

**Comment by danielyli** [comment]
+1 for this feature.

I agree with @aishfenton that Matrix constructor as currently implemented is a little counterintuitive, since everywhere else is column major.  Maybe add a `@deprecated("in Breeze 1.0, this will be a column-major constructor; use Matrix(v1.t, v2.t, ...) to future-proof your code")` to give people plenty of time to know of the new constructor?
